recipes/limix-core/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "limix-core" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "b32d92b1c5294b112028eaa75cf00197257850a7f065533f0a907915d934f34c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - numpy >=1.10
+    - scipy >=0.18
+    - six
+
+test:
+  requires:
+    - pytest >=3
+  imports:
+    - limix_core
+  commands:
+    - python -c "import sys; import limix_core; sys.exit(limix_core.test())"
+
+about:
+  home: https://github.com/limix/limix-core
+  license: MIT
```

Comment:
It appears as if the license should be `APACHE-2.0`:

https://github.com/limix/limix-core/blob/master/LICENSE